website/pages/docs/troubleshooting.md

```diff
@@ -22,3 +22,21 @@ There are a couple of ways to get help for any CloudQuery-related issues or ques
 ### Verbose Logging
 
 Usually the first step that will be needed to debug/resolve an issue is to run `cloudquery` with `--log-level debug` to enable verbose logging.
+
+### Error: "failed to migrate source"...
+
+If you see an error such as `failed to migrate source`, it means that, while upgrading a plugin, the migration of the SQL schema failed.
+CloudQuery makes a best-effort to automatically and transparently manage the schema of the plugins, but 
+this can sometimes fail during version upgrades.
+
+The solution is to drop-and recreate the database or schema. (or all the plugins tables, such as `aws_*`).
+Some queries that may help you (handle with care!):
+
+```sql
+DROP SCHEMA public CASCADE;
+CREATE SCHEMA public;
+```
+
+```sql
+\dt aws_*
+```
```

Comment:
We can maybe also mention here that there are other, less destructive, solutions. However, this will usually involve inspecting the schema changes (as described in the changelog) and manually migrating them.

Reply:
Not sure we want to have that here - since if the user gets here it means that our auto-migration already failed (but only on some of the tables probably)- and something is broken in a non-trivial way..

Reply:
Yeah, for sure, it will be non-trivial, and we'll try and make it as rare as possible. But perhaps it's a worthwhile option for some users who have a lot of triggers set up around their database.